lib/server.js

```diff
@@ -15,7 +15,9 @@
 
 'use strict';
 
-var _ = require('lodash');
+var isString = require('lodash/isString');
```

Comment:
@gabrielcsapo On the server, it can be a minor savings of space and time, depending on the complexity of the dependency graph. If you're using node-inspector to debug, it's good to load less unused code, since the inspector can bog down quickly. It's on the client where it's crucial, to save on bundle size.